packages/wrangler/src/inspect.ts

```diff
@@ -596,7 +606,12 @@ export default function useInspector(props: InspectorProps) {
 			try {
 				// Intercept Debugger.scriptParsed responses to inject URL schemes
 				const message = JSON.parse(event.data as string);
-				if (message.method === "Debugger.scriptParsed") {
+				if (
+					message.method === "Debugger.scriptParsed" &&
+					// Breakpoint debugging doesn't work (breakpoints can be set, but not hit) with the worker:// scheme, so
+					// disable in local mode.
+					!localMode
```

Comment:
I don't think we should be checking for local mode here. This section should also work in remote mode to support sourcemaps